pyartcd/pyartcd/pipelines/ocp4_scan.py

```diff
@@ -64,7 +64,7 @@ async def run(self):
             # Inconsistency probably means partial failure and we would like to retry.
             #  but don't kick off more if already in progress.
             self.logger.info('Triggering a %s RHCOS build for consistency', self.version)
-            await trigger_rhcos(self.version, True)
+            jenkins.start_rhcos(self.version, True)
```

Comment:
Would make blocking a keyword param to make it more clear

Reply:
Done in 12fb9f899a3c5950ba6c997eb78d4cfe8da52cf8